File: 1480/runningSum.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        returnArray = []
        sum = 0
        for numbers in nums:
            sum += numbers
            returnArray.append(sum)
        return returnArray

File: 1480/runningSum.java
class Solution {
    public int[] runningSum(int[] nums) {
        int resultingArray[] = new int[nums.length];
        int sum = 0;
        for(int i = 0; i < nums.length; i++)
        {
            sum += nums[i];
            resultingArray[i] = sum;
        }

        return resultingArray;
    }
}